did-pkh/Cargo.toml

```diff
@@ -12,10 +12,12 @@ homepage = "https://github.com/spruceid/ssi/tree/main/did-pkh/"
 documentation = "https://docs.rs/did-pkh/"
 
 [features]
-default = ["ssi/ring"]
 
 [dependencies]
-ssi = { version = "0.4", path = "../", default-features = false, features = ["secp256k1", "keccak-hash", "secp256r1", "ripemd160"] }
+ssi-caips = { path = "../ssi-caips", version = "0.1", features = ["eip", "ripemd-160"] }
+ssi-dids = { path = "../ssi-dids", version = "0.1"}
+ssi-jwk = { path = "../ssi-jwk", features = ["ripemd-160", "eip"] }
+ssi-crypto = { path = "../ssi-crypto", version = "0.1"}
```

Comment:
If I am just importing `did-X`, e.g. `did-pkh`, how do I pick the crypto backend? E.g. rust-crypto vs ring?

Reply:
I wasn't sure if the did methods needed the same level of feature selection, I found that in practice they would almost always be imported alongside their dependancies, which would [unify the features](https://doc.rust-lang.org/cargo/reference/features.html#feature-unification). E.g. `ssi-dids = { ... features = [x, y, z] }` would apply `x, y, z` to the `ssi-dids` dependancy of `did-pkh`.

Reply:
In that case should `ssi/ring` be a default feature for the did-methods?